impeded tickets will now appear on projectboardchecks, make_venv.bat …

danielmaclaren opened the pull request:
…added and README.md updated

danielmaclaren committed:
impeded tickets will now appear on projectboardchecks, make_venv.bat added and README.md updated

danielmaclaren commented:
I've rewritten the readme.md file, which should be used for setting up and creating the venv for testing purposes.

Output should look like this screenshot:

![image](https://github.com/ISISComputingGroup/ProjectBoardUtils/assets/101112838/857fb825-85e7-4456-8c8a-e424b297eab3)

LowriJenkins commented on card.py:
```diff
@@ -196,6 +196,7 @@ def check_if_stale(issue, label_name, warn_days_allowed, error_days_allowed, ass
             #     check_column_label(labels, 'completed', issue)
             if column is COLUMNS.IMPEDED:
                 check_column_label(labels, 'impeded', issue)
+                check_if_stale(issue, 'impeded', 7, 28, assigned)
```
does this actually check if the ticket has been updated by a comment as was suggested in planning? or have some other way of checking that the ticket has been acknowleged? It appears that this check only takes into account creation time and whether a ticket has moved recently, which does not fulfil the acceptence criteria of having a way to acknowledge that the ticket is _still_ impeded.

zsoltkebel replied:
Has this been resolved?

danielmaclaren committed:
added most recent comment check to the impeded tickets

isaachilly committed:
Removed printing out of whole comment & changed comment time to be when it was last edited.

If comment was long this clogged up the log and comment may be old and not useful for checking, Also changed comment time to be when it was last edited, as we want to check last time someone looked at/checked up on/thought about the issue.
